src/context/Iterator.cpp

```diff
@@ -579,7 +574,12 @@ List PropIter::getVertices() {
 List PropIter::getEdges() {
     DCHECK(iter_ == rows_.begin());
     List edges;
+    edges.values.reserve(size());
     for (; valid(); next()) {
+        auto edge = getEdge();
+        if (edge.isEdge()) {
+            const_cast<Edge&>(edge.getEdge()).format();
+        }
         edges.values.emplace_back(getEdge());
```

Comment:
edges.values.emplace_back(std::move(edge));
and put it into if

Reply:
Null is also acceptable.

Reply:
I think what ding means is that you need not to call `getEdge()` again since `edge` has been there.